evm_loader/utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Permissions management scripts
+
+Scripts mint_permission_token.sh mint_many.sh are suited to operate with client's and contract's permission token balances - to grant or deprive permissions.
+
+## Requirements
+
+To use this scripts one should have next CLI utilities installed:
+    - solana CLI interface
+    - neon-cli 
```

Comment:
I noticed that we don't even have any install/usage instruction for neon-cli.
Alexander N. confirms this, saying also what he'd like to add neon-cli to our docs

It's not an issue of this PR, just be aware

Reply:
I believe, some day we will publish it in [crates.io](https://crates.io/)